_pages/docs/tutorials/beginner-tutorial.mdx

```diff
@@ -0,0 +1,213 @@
+    * Go to your **Applications/Utilites** folder. Double-click **Terminal**.
+* Once the **Terminal** is open, you will need to navigate to the folder in which you put all the files. As an example, let's say you put them on the Desktop. You would type the following in your **Terminal** and hit "return":
+
+```    
+cd ~/Desktop
```

Comment:
Hard to explain in text, but this kinda thing can make the terminal a little easier for some folk.
Just a suggestion.

https://user-images.githubusercontent.com/29681384/205765129-bf03cb8e-ea2f-4027-a433-666dab7cef3f.mov